<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1076</width>
    <height>496</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="tabShape">
       <enum>QTabWidget::Triangular</enum>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <property name="elideMode">
       <enum>Qt::ElideNone</enum>
      </property>
      <property name="documentMode">
       <bool>false</bool>
      </property>
      <property name="tabsClosable">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Вход</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="3" column="1" colspan="3">
         <widget class="QPushButton" name="doLogin">
          <property name="text">
           <string>Вход</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="label">
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>Логин</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2" colspan="2">
         <widget class="QLineEdit" name="userName"/>
        </item>
        <item row="0" column="2">
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="2" column="1">
         <widget class="QLabel" name="label_2">
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>Пароль</string>
          </property>
         </widget>
        </item>
        <item row="2" column="2" colspan="2">
         <widget class="QLineEdit" name="password">
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>0</height>
           </size>
          </property>
          <property name="echoMode">
           <enum>QLineEdit::Password</enum>
          </property>
         </widget>
        </item>
        <item row="5" column="2">
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="2" column="0">
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="2" column="4">
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="4" column="1" colspan="3">
         <widget class="QPushButton" name="setAdminPasswd">
          <property name="text">
           <string>Создать пароль администратора</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string> Управление учетными записями</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="1" column="0">
         <widget class="QLabel" name="newUserName_label">
          <property name="text">
           <string>Имя пользователя</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0" colspan="7">
         <widget class="QTableWidget" name="usersTable"/>
        </item>
        <item row="1" column="1" colspan="2">
         <widget class="QLineEdit" name="newUserName"/>
        </item>
        <item row="1" column="6">
         <widget class="QPushButton" name="add">
          <property name="text">
           <string>Добавить</string>
          </property>
         </widget>
        </item>
        <item row="1" column="3" colspan="2">
         <widget class="QLabel" name="newPassword_label">
          <property name="text">
           <string>Пароль</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0" colspan="7">
         <widget class="QLabel" name="warning">
          <property name="font">
           <font>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Требуется вход с правами администратора</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="1" column="5">
         <widget class="QLineEdit" name="newPassword"/>
        </item>
        <item row="2" column="1" colspan="6">
         <widget class="QComboBox" name="workerList"/>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="workerList_label">
          <property name="text">
           <string>Владелец:</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Управление списком сотрудников</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="8" column="0">
         <widget class="QLabel" name="salesWorkersTable_label">
          <property name="text">
           <string>Sales:</string>
          </property>
         </widget>
        </item>
        <item row="6" column="0">
         <widget class="QLabel" name="manageWorkersTable_label">
          <property name="text">
           <string>Manage:</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="EmployeeWorkersTable_label">
          <property name="text">
           <string>Employee:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="3" rowspan="2" colspan="2">
         <widget class="QDateEdit" name="date">
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="1" column="5" rowspan="2">
         <widget class="QLabel" name="basePay_label">
          <property name="text">
           <string>Базовая ставка</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="workerName">
          <property name="maximumSize">
           <size>
            <width>200</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="3" column="5">
         <widget class="QLabel" name="newUserPasswordPM_label">
          <property name="text">
           <string> Пароль</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="workerName_label">
          <property name="text">
           <string>ФИО</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0" colspan="2">
         <widget class="QCheckBox" name="createUserAcc">
          <property name="text">
           <string>Создать учетную запись</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2" rowspan="2">
         <widget class="QLabel" name="date_label">
          <property name="text">
           <string>Дата поступления</string>
          </property>
         </widget>
        </item>
        <item row="1" column="12">
         <widget class="QComboBox" name="workerType">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>0</height>
           </size>
          </property>
          <item>
           <property name="text">
            <string>Employee</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Manager</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Sales</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="3" column="12">
         <widget class="QPushButton" name="deleteWorker">
          <property name="text">
           <string>Удалить</string>
          </property>
         </widget>
        </item>
        <item row="1" column="9">
         <widget class="QLabel" name="workerType_label">
          <property name="text">
           <string>Должность</string>
          </property>
         </widget>
        </item>
        <item row="3" column="6">
         <widget class="QLineEdit" name="newUserPasswordPM">
          <property name="maximumSize">
           <size>
            <width>200</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="3" column="3">
         <widget class="QLineEdit" name="newUserNamePM">
          <property name="maximumSize">
           <size>
            <width>200</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="3" column="2">
         <widget class="QLabel" name="newUserNamePM_label">
          <property name="text">
           <string>Имя учетной записи</string>
          </property>
         </widget>
        </item>
        <item row="3" column="9">
         <widget class="QPushButton" name="addWorker">
          <property name="text">
           <string>Добавить</string>
          </property>
         </widget>
        </item>
        <item row="1" column="14">
         <widget class="QComboBox" name="master">
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="1" column="6" rowspan="2" colspan="3">
         <widget class="QLineEdit" name="basePay">
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="1" column="13">
         <widget class="QLabel" name="master_label">
          <property name="text">
           <string>Начальник</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0" colspan="15">
         <widget class="QLabel" name="warning_2">
          <property name="font">
           <font>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Требуется вход с правами администратора</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="1" column="15">
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="5" column="0" colspan="16">
         <widget class="QTableWidget" name="employeeWorkersTable">
          <column>
           <property name="text">
            <string>Имя</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Дата поступления</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Базовая ставка</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Начальник</string>
           </property>
          </column>
         </widget>
        </item>
        <item row="7" column="0" colspan="16">
         <widget class="QTableWidget" name="manageWorkersTable">
          <column>
           <property name="text">
            <string>Имя</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Дата поступления</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Базовая ставка</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Начальник</string>
           </property>
          </column>
         </widget>
        </item>
        <item row="9" column="0" colspan="16">
         <widget class="QTableWidget" name="salesWorkersTable">
          <column>
           <property name="text">
            <string>Имя</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Дата поступления</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Базовая ставка</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Начальник</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>Моя страница</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="4" column="0" colspan="3">
         <widget class="QTableWidget" name="subordinatesTable">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>12312312</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="subordinates_label">
          <property name="text">
           <string>Подчиненные:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0" rowspan="2">
         <widget class="QLabel" name="currentCash">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>200</width>
            <height>20</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>8</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Заработная плата на текущий момент:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
          </property>
         </widget>
        </item>
        <item row="1" column="1" rowspan="2">
         <widget class="QLabel" name="currentPayment">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>20</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
          </property>
         </widget>
        </item>
        <item row="0" column="0" colspan="3">
         <widget class="QLabel" name="warning_3">
          <property name="font">
           <font>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Требуется вход с учетной записью сотрудника</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1076</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
